services/account/controllers/AccountController.ts

```diff
@@ -10,7 +11,30 @@ export class AccountController {
     this.repository = new AccountRepository();
   }
 
-  async getAccount(accountNumber) {
-    return await this.repository.find(accountNumber);
+  async getAccount(filter) {
+    try {
```

Comment:
The ```try/catch``` is for ```JSON.parse(filter)``` because filter is being passed in as a string and needs to be converted to an object. It's not for the promise.